### DISCUSSION TITLE
The start page has broken

### DISCUSSION THREAD

**Discussion by DarkTraveler** [strapi/strapi#5845] [Legacy Topics]
Hey.

**Describe the bug**
My start page does not work. I observe this problem on my laptop and on the customer's VDS.
Аll other pages are not broken.
![image](https://user-images.githubusercontent.com/16899697/79553757-54165200-80e0-11ea-9b3c-4dfaec16dcf9.png)
**Steps to reproduce the behavior**

1. delete .cache and build folders
2. update strapi to version beta.20
3. run strapi

**System**
nodejs v12.16.1
NPM 6.13.4
Strapi version: 3.0.0-beta.20
Database: mongodb
Operating system: WSL Windows 10 - Ubuntu 18.04

**Comment by MattieBelt** [comment]
@DarkWayfarer Did you migrate from 19.x? There were some changes made regarding the index page. Follow the [(migration) docs](https://strapi.io/documentation/3.0.0-beta.x/migration-guide/migration-guide-beta.19-to-beta.20.html#index-page)

**Comment by DarkTraveler** [comment]
@MattieBelt 

Hey. thanks a lot. I really forgot this time to check the migration section ... But there was 1 more problem that I did not manage to write about. I cannot delete an already attached image. Do you also? If so, then I will create an issue.

Database Mongo

![image](https://user-images.githubusercontent.com/16899697/79691227-989d1b80-82a1-11ea-9231-86ba7c48911c.png)

**Reply by MattieBelt** [reply]
@DarkWayfarer No problem. 

No I'm not familiar with this problem, haven't had the time to completely try/explore the new update. 
Also I'm not using Mongo, so I didn't have to migrate my mongoDB to beta.20. Maybe something went wrong, because you already run it with beta.20 packages before following [the guide](https://portal.productboard.com/strapi/1-public-roadmap/c/66-sdk-js). Maybe rollback and migrate again.

**Comment by DarkTraveler** [comment]
@MattieBelt Hey. 
Thanks for discussing the issue. 
Unfortunately, I did the rollback of the database primarily before the migration. Today I will check on a clean project and do Issues.
